solidity/random-beacon/contracts/test/TestToken.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Maybe we don't have to commit this file in this PR? It depends on how much would you like to get configured in this PR. If we will have this sample contract it would be nice to add a deployment script and a unit test samples as well.

Reply:
Yea, I figured I'd add a sample contract plus test and a deployment script. It will set up a path for development. Whoever comes next can spend less time on configuration work.

Reply:
Awesome! Super helpful! We just need to remember to remove these files once we replace them with the proper ones. Maybe we should add a comment to each file that is a stub so we remember to replace them with the proper implementations?

Reply:
Sure, added comments 61b678d